src/DocChecks.jl

```diff
@@ -205,9 +208,12 @@ function linkcheck(link::Markdown.Link, doc::Documents.Document)
                 push!(doc.internal.errors, :linkcheck)
                 @error "linkcheck '$(link.url)' status: $(status)."
             end
+        elseif occursin(FTP_STATUS_REGEX, result)
+            # this regex is matched iff success
+            printstyled(INDENT, "FTP SUCCESS ", link.url, "\n", color=:green)
```

Comment:
`INDENT` is missing. Also, it should be consistent with the `HTTP` success and use an `@debug`.